container/example-config/static-only.toml

```diff
@@ -13,3 +13,13 @@ hash = "Qm000"
 id = "ui-interface-1"
 bundle = "bundle1"
 port = 3000
+dna_interface = "http interface"
+
+
+[[interfaces]]
+admin = true
+id = "http interface"
+
+[interfaces.driver]
+port = 4000
+type = "http"
```

Comment:
Since hc-web-client can only talk websocket and since this is mainly catered for that use case, would only use websocket interfaces as examples here..